NewDawn/NewDawn/ViewControllers/SettingPageViewControllers/EditProfileSubViewController.swift

```diff
@@ -645,9 +645,9 @@ class EditProfile_Smoke: UIViewController {
 
 class EditProfile_Drink: UIViewController {
     
-    let FREQUENT_DRINK = "frequent"
-    let NODRINK = "no"
-    let SOCIAL_DRINK = "social"
+    let FREQUENT_DRINK = "经常喝"
```

Comment:
I just have another PR which unified these with registration page. Maybe you can leave as it and use my PR for this change?

Reply:
Yea I just saw it.

Reply:
You can push yours, and I will pull from master.